frontend/src/App.tsx

```diff
@@ -8,6 +8,7 @@ import { GlobalStyle } from "./GlobalStyle";
 import { Root } from "./layout/Root";
 import { RealmPage } from "./page/Realm";
 import { HomePage } from "./page/Home";
+import { PlayerPage } from "./page/Player";
```

Comment:
I would rather say it's a `VideoPage` or `EventPage`. Same applies to all other instances of "player".

Reply:
We went for `Video`